tests/e2e/initialization/config.go

```diff
@@ -295,9 +295,6 @@ func updateBankGenesis(bankGenState *banktypes.GenesisState) {
 			},
 		},
 	})
-	if len(bankGenState.SupplyOffsets) == 0 {
-		bankGenState.SupplyOffsets = []banktypes.GenesisSupplyOffset{}
-	}
```

Comment:
Is this something fixed with upstream changes?

Reply:
That was needed before https://github.com/osmosis-labs/osmosis/pull/2308 got merged, for some reason